client-go/controller/models/apps/apps_test.go

```diff
@@ -291,7 +291,7 @@ func TestAppsList(t *testing.T) {
 
 	client := client.Client{HTTPClient: httpClient, ControllerURL: *u, Token: "abc"}
 
-	actual, err := List(&client)
+	actual, err := List(&client, 100)
```

Comment:
I feel like this [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29) should be part of the client config with a default of 100. That way users can tune this number by modifying ~/.deis/client.json